src/app/utils/is-nullable.ts

```diff
@@ -0,0 +1,7 @@
+export function isNullable(value: unknown): value is null | undefined {
+  return value == null;
+}
+
+export function isNotNullable<T>(value: T): value is NonNullable<T> {
+  return value != null;
```

Comment:
Return ```true``` if the value is not null or undefined, false otherwise.
```suggestion
  return value !== null && value !== undefined;
```